bundles/org.openhab.binding.feed/src/main/java/org/openhab/binding/feed/internal/handler/FeedHandler.java

```diff
@@ -59,10 +58,15 @@
  */
 public class FeedHandler extends BaseThingHandler {
 
+    private final ChannelGroupUID titleChannelGroupUID = new ChannelGroupUID(getThing().getUID(), "Titles");
+    private final ChannelGroupUID descriptionChannelGroupUID = new ChannelGroupUID(getThing().getUID(), "Descriptions");
+    private final ChannelGroupUID dateChannelGroupUID = new ChannelGroupUID(getThing().getUID(), "Dates");
+
     private Logger logger = LoggerFactory.getLogger(FeedHandler.class);
 
     private String urlString;
     private BigDecimal refreshTime;
+    private BigDecimal numberOfEntries;
```

Comment:
Is there a special reason why this is a `BigDecimal`? Usually primitives ('int`) are preferred.

(also applies to `refreshTime`, but you don't need to change that).